content/master/concepts/patch-and-transform.md

```diff
@@ -1654,6 +1655,25 @@ patches:
           trim: `-north-1'
 ```
 
+#### Join
```

Comment:
Can you move this section to be before line 1601, the heading "Regular expression type" to make the type order alphabetical?

Reply:
Done.